packages/veritone-react-common/src/components/GeoPicker/index.js

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import {
+  oneOf,
+  oneOfType,
+  shape,
+  func,
+  bool,
+  string,
+  number
+} from 'prop-types';
+
+import { CircularProgress } from 'material-ui/Progress';
+import Typography from 'material-ui/Typography';
+import GoogleMapsLoader from 'google-maps';
+
+import styles from './styles.scss';
+
+/* eslint-disable no-undef*/
```

Comment:
not immediately obvious to me what this is fixing -- can it be removed, or just a disable-line?